VCFv4.3.tex

```diff
@@ -1277,7 +1288,7 @@ \subsection{Overall file organization}
 No record may refer to a contig not present in the header itself.
 
 \item All INFO and GENOTYPE fields must be fully typed in the BCF2 header to
-enable type-specific encoding of the fields in records.  An error should be
+enable type-specific encoding of the fields in records.  An error must be
```

Comment:
This is not about the standard, really. BCF must know all fields at the time of writing the header.